fuikit/Classes/Core/FPUICollectionViewDatasource.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  FPUICollectionViewDatasource.swift
+//  fuikit
+//
+//  Created by Remmington Damper on 1/25/22.
+//
+
+import Foundation
+
+@objc open class FPUICollectionViewDatasource: NSObject, UICollectionViewDataSource, UIScrollViewDelegate {
```

Comment:
Once moved, delete the `UIScrollViewDelegate` and capitalize the 's' in 'Datasource'
```suggestion
@objc open class FPUICollectionViewDataSource: UICollectionViewDataSource {
```